component-library/component-lib/src/lib/banner/banner.component.ts

```diff
@@ -62,14 +62,15 @@ export class BannerComponent implements OnInit {
   };
 
   eventHandler(emitValue: string){
+    console.log(emitValue);
```

Comment:
Do we need this console log?

Reply:
Added for Keyur to test. Since eventHandler's only functionalit is emit I'm logging the emitvalue.